### PR TITLE
[137] Product Page using HTML

### PR DESCRIPTION
**Pull request by zao12345**
webpage 1
webpage 2

### CODE REVIEW

File: submissions/ZainabAbdussalam-Osimade/HTML/Product.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <p> Available in Wine, Camo, and Black </p>
+  
+            <p>
+            <b>Feature</b>
+            </p>
+
+            <p>Giving you the desired warmth. You can go anywhere, do anything with this puffer jacket. <br/> This on-trend coat compresses and packs in its own pocket/bag, making it the perfect traveling companion. </p>
+    
+            <p>Stand collar with fleece lining; detachable hood <br/> Zip-front closure <br/> Two front zipper pockets <br/> Allover quilted pattern <br/> Wind resistant <br/> Compresses and folds into its own pocket for easy packing <br/> Solid shell: 100% Nylon; Lining: 100% Polyester; Print shell and lining: 100% Polyester <br/> Machine washable </p>
+
```
Comment (Ireti203) [inline submissions/ZainabAbdussalam-Osimade/HTML/Product.html:26]: Your code is not properly formatted. You do not need to leave a line after every  line of code, kindly remove whitespaces. The body tag is not properly formatted.

File: submissions/ZainabAbdussalam-Osimade/HTML/Product.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            <img src="Images/2 image.jpg" width="180px"/>
+
+            <p>
+                 <b>Fam Collection Women's Belted Quilted Jacket With Hood </b> <br/> <b> #30,000 </b>
+            </p>
+           
+            <p> Available in White, Brown, and Navy Blue </p>
+  
+            <p>
+            <b>Feature</b>
```
Comment (Ireti203) [inline submissions/ZainabAbdussalam-Osimade/HTML/Product.html:42]: KIndly format properly

File: submissions/ZainabAbdussalam-Osimade/HTML/Product.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+               
+            </p>
+
+            <p>
+                    �FamFashionHub2020
+
+            </p>
+
+
+</body>
```
Comment (Ireti203) [inline submissions/ZainabAbdussalam-Osimade/HTML/Product.html:67]: Same here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove spaces on image name. Use `camelCase`, `PascalCase`, `snake_case` or `kebab-case` for naming thing

**Review by Ireti203 — APPROVED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**